composer.json

```diff
@@ -12,7 +12,6 @@
     "minimum-stability": "stable",
     "require": {
         "php": ">=5.5",
-        "equip/adr": "^2.0",
```

Comment:
Independent of this PR being merged, are we designating equip/adr as deprecated on Packagist?